interview-questions.md

```diff
@@ -7,53 +7,53 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. Are (most) Ruby methods accessors or mutators?
 
-  Your answer:
+  Your answer: Most ruby methods are accessors, meaning they do not change the original array whereas mutators will completely modify the input variable. However, the bang operator can be used with a method to make it a mutator method.  
 
-  Researched answer:
+  Researched answer: Ruby accessor methods return the orginal variables' values. Ruby's accessor methods consist of attr_reader, attr_writer, and attr_accessor. Using these accessor methods, references the behavior of the instance variable only once; therefore, it is no longer referenced directly throughout the program. Attr_reader allows you to get the value of the instance variable, attr_writer allows you to set the value, and attr_accessor allows you to do both. Attr_accessor is a shortcut to set up getter and setter methods in Ruby. It is a way to set and get values from outside the class. 
```

Comment:
Very close to the asked question! But this si still a great write up for the "attr_" class of methods